### PR TITLE
SC: Add API for operatorThreshold

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR added an API for operatorThreshold.

```
> subbridge.getValueTransferOperatorThreshold("0xbed205fb9d6e2a69ad1d25a428a08fdd9f2a2c2c")
2
> subbridge.setValueTransferOperatorThreshold("0xbed205fb9d6e2a69ad1d25a428a08fdd9f2a2c2c",5)
"0x0538ddb4da8f1752e4696d260beddad8dfff065b5eb1b1cbbd466bf9857fb1cb"
> subbridge.getValueTransferOperatorThreshold("0xbed205fb9d6e2a69ad1d25a428a08fdd9f2a2c2c")
5
```

```
> subbridge.setValueTransferOperatorThreshold("0xbed205fb9d6e2a69ad1d25a428a08fdd9f2a2c2b",5)
Error: bridge does not exist
    at web3.js:3276:20
    at web3.js:6449:15
    at web3.js:5214:36
    at <anonymous>:1:1

> subbridge.getValueTransferOperatorThreshold("0xbed205fb9d6e2a69ad1d25a428a08fdd9f2a2c2a")
Error: bridge does not exist
    at web3.js:3276:20
    at web3.js:6449:15
    at web3.js:5214:36
    at <anonymous>:1:1

```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/sc/bridge_manager.go [hunk 1]
```diff
@@ -734,6 +734,40 @@ func (bm *BridgeManager) stopAllRecoveries() {
...
+	bi, exist := bm.GetBridgeInfo(bridgeAddr)
+
+	if !exist {
+		return common.Hash{}, errors.New("bridge does not exist")
+	}
+
+	bi.account.Lock()
+	tx, err := bi.bridge.SetOperatorThreshold(bi.account.GetTransactOpts(), voteTypeValueTransfer, threshold)
+	if err != nil {
+		bi.account.UnLock()
```
Comment (jeonghonah) [inline node/sc/bridge_manager.go]: how about simply using `defer` statement?

File: node/sc/bridge_manager.go [hunk 2]
```diff
@@ -734,6 +734,40 @@ func (bm *BridgeManager) stopAllRecoveries() {
...
+	bi.account.UnLock()
+
+	return tx.Hash(), nil
+}
+
+func (bm *BridgeManager) GetValueTransferOperatorThreshold(bridgeAddr common.Address) (uint8, error) {
+	bi, exist := bm.GetBridgeInfo(bridgeAddr)
+
+	if !exist {
+		return 0, errors.New("bridge does not exist")
```
Comment (jeonghonah) [inline node/sc/bridge_manager.go]: We can declare an error variable for the duplicated messages.

File: node/sc/bridge_manager.go (hunk 1 above)
Comment (KimKyungup) [reply node/sc/bridge_manager.go]: reflected. :)

File: node/sc/bridge_manager.go (hunk 2 above)
Comment (KimKyungup) [reply node/sc/bridge_manager.go]: reflected. :)

**Pull request merged**